app/docs/development.rst

```diff
@@ -18,16 +18,21 @@ Installation
 Please ensure that the correct backend is enabled in your docker settings, and run all of the following commands in the ``wsl`` shell.
 At the time of writing, we use ``Ubuntu 20.04`` from the Microsoft store as the default distro.
 As WSL2 is slow at syncing files between Windows and WSL2 filesystems it is best to checkout the codebase within ``wsl`` itself.
-For the same reason, we currently recommend running PyCharm from wsl rather than from windows (see `Running WSL GUI Apps on Windows 10`_), but improved support for this should follow from JetBrains in 2021.
 
-2. Clone the repo
+2. (Optional) Download and install buildx for Docker
+
+The docker-compose cycle script below utilizes `Docker Buildx`_. Depending on the steps above, buildx should be
```

Comment:
Could you move this to the "Download and install docker" step? It's not optional, unfortunately.

Reply:
Done. As a side note: the docs linked refer to it being installed by default in the packages. Hence the optionality being added.